OpenRA.Game/World.cs

```diff
@@ -356,24 +378,20 @@ public int SyncHash()
 				var n = 0;
 				var ret = 0;
 
-				// hash all the actors
+				// Hash all the actors.
 				foreach (var a in Actors)
 					ret += n++ * (int)(1 + a.ActorID) * Sync.HashActor(a);
 
-				// hash all the traits that tick
+				// Hash all the traits that tick.
```

Comment:
Since you've already touched this, can you change it to be correct or just remove it?
It has been a very long time since this has acted on the traits that tick.